src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,12 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: 'Under Construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+});
```

Comment:
- [ ] You're meant to create a  reducer that checks the status and always returns "Under construction" (the initial state should check to that string).
- Kindly creates 1 reducer and call it  `checkStatus`, then you can export by default `categoriesSlice.reducer` and  export  `checkStatus` as an action

## Sample code:

```
export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

## Instructions :
![image](https://github.com/abiola-arobieke/book-store/assets/50990925/a83cf8d1-7d88-4f36-9819-49794c541618)